.github/workflows/docker.yml

```
@@ -0,0 +1,25 @@
+name: Build CLI Docker image
+
+on:
+  push:
+    paths:
+      - 'Dockerfile'
```

```suggestion
      - './Dockerfile'
```

Otherwise we going to match files in the build folder